pages/account/manage/event/[[...data]].js

```diff
@@ -69,6 +70,14 @@ export default function ManageEvent({ BASE_URL, event }) {
   const [endDate, setEndDate] = useState(
     event.date?.end && formatDate(event.date?.end)
   );
+  const DropDownOptions = ["Non-speaking","Speaking"];
```

Comment:
Duplicate again, please set in config and reuse

Reply:
Removed duplicacy:
```
  const [endDate, setEndDate] = useState(
    event.date?.end && formatDate(event.date?.end)
  );
  const DropDownOptions = config.events.userStatus;
```